setup.py

```diff
@@ -128,6 +128,7 @@
     "torchvision",
     "transformers>=4.25.1",
     "urllib3<=2.0.0",
+    "peft>=0.5.0"
```

Comment:
I added it here so that the CI will automatically fall back to PEFT tests once we make a new release on PEFT. I can also remove that and make a PR on diffusers once we make a release. 
In both cases after the PEFT release we'll have to update that line to use PEFT >= 0.6.0